modules/peridynamics/src/auxkernels/BondStatusPD.C

```diff
@@ -20,7 +20,7 @@ validParams<BondStatusPD>()
   params.addClassDescription("Class for updating the bond status based on different failure "
                              "criteria: critical stretch and "
                              "maximum principal stress");
-  MooseEnum FailureCriteriaType("Critical_Stretch Maximum_Tensile_Stress", "Critical_Stretch");
```

Comment:
A comment here:  I changed these names to be more consistent with MOOSE conventions.  I expected to break a test when I did that, but this is apparently not tested.